src/test/java/de/numcodex/sq2cql/TranslatorTest.java

```diff
@@ -373,34 +384,298 @@ void toCQL_GeccoTask2() {
+                                "display": "MDAT speichern/verarbeiten",
+                                "system": "urn:oid:2.16.840.1.113883.3.1937.777.24.5.3"
+                            }
+                        ]
+                    },
+                    {
+                        "fhirPath": "provision.provision.code",
+                        "searchParameter": "mii-provision-provision-code",
+                        "type": "coding",
+                        "value": [
+                            {
+                                "code": "2.16.840.1.113883.3.1937.777.24.5.3.8",
+                                "display": "MDAT wissenschaftlich nutzen EU DSGVO NIVEAU",
+                                "system": "urn:oid:2.16.840.1.113883.3.1937.777.24.5.3"
+                            }
+                        ]
+                    },
+                    {
+                        "fhirPath": "provision.provision.code",
+                        "searchParameter": "mii-provision-provision-code",
+                        "type": "coding",
+                        "value": [
+                            {
+                                "code": "2.16.840.1.113883.3.1937.777.24.5.3.9",
+                                "display": "MDAT zusammenfuehren mit Dritte",
+                                "system": "urn:oid:2.16.840.1.113883.3.1937.777.24.5.3"
+                            }
+                        ]
+                    },
+                    {
+                        "fhirPath": "provision.provision.code",
+                        "searchParameter": "mii-provision-provision-code",
+                        "type": "coding",
+                        "value": [
+                            {
+                                "code": "2.16.840.1.113883.3.1937.777.24.5.3.1",
+                                "display": "PATDAT erheben/speichern/nutzen",
+                                "system": "urn:oid:2.16.840.1.113883.3.1937.777.24.5.3"
+                            }
+                        ]
+                    },
+                    {
+                        "fhirPath": "provision.provision.code",
+                        "searchParameter": "mii-provision-provision-code",
+                        "type": "coding",
+                        "value": [
+                            {
+                                "code": "2.16.840.1.113883.3.1937.777.24.5.3.26",
+                                "display": "Rekontaktierung/Ergaenzungen",
+                                "system": "urn:oid:2.16.840.1.113883.3.1937.777.24.5.3"
+                            }
+                        ]
+                    }
+                ],
+                "key": {
+                    "code": "combined-consent",
+                    "display": "Einwilligung f\\u00fcr die zentrale Datenanalyse",
+                    "system": "mii.abide"
+                },
+                "primaryCode": {
```

Comment:
Another possible name for `primaryCode` would be `mappedKey`, but I'm fine with `primaryCode` as long as it is documented properly.

Reply:
I think primaryCode is the way to go as it indicates its relation to the primaryCodePath defined in CQL. We should decouple the key more from the mapping moving forward.